### ISSUE TITLE
Android Nougat errors

### ISSUE THREAD

**Issue by lightydo** [xmikos/setools-android#9]
Copied setools to Nexus 5X with Android Nougat (NRD90S). TWRP present, SU present. 
bullhead: # seinfo
ERROR: more than one specifier
ERROR: failed on entry 24 of 10973
ERROR: Unable to open policy /sepolicy.
ERROR: Success

**Comment by lightydo** [comment]
I pulled the sepolicy file into host and tried to run "seinfo --all sepolicy" which gave me: 

ERROR: policydb version 30 does not match my version range 15-29
ERROR: Unable to open policy sepolicy.
ERROR: Success

**Comment by Surge1223** [comment]
The errors for Nougat is because the policydb define for policy version range  libsepol are out of date compared to the aosp and setools3 versions or the file.  You could probabaly just grep libsepol dot for the policy version and change the range and have it working

**Comment by pasis** [comment]
The problem from the first post is discussed in http://thread.gmane.org/gmane.comp.security.seandroid/2824 .

I'm going to fix it, but I'm quite confused by difference of upstream and aosp libsepol. Both support version 30, but it seems binary formats differ. Also upstream version changed API and setools3 (that are used in setools-android) don't build against it. So, fix requires some time effort.

**Comment by Surge1223** [comment]
I'm using libsepol from master,  checkpolicy seems to be able to parse 

```
angler:/ # checkpolicy /sepolicy -b -d -M -c 30 -o /sdcard/sepolicy.new
checkpolicy:  loading policy configuration from /sepolicy
libsepol.policydb_index_others: security:  1 users, 2 roles, 801 types, 0 bools
libsepol.policydb_index_others: security: 1 sens, 1024 cats
libsepol.policydb_index_others: security:  63 classes, 13466 rules, 0 cond rules
checkpolicy:  policy configuration loaded
checkpolicy:  writing binary representation (version 30) to /sdcard/sepolicy.new

Select an option:
0)  Call compute_access_vector
1)  Call sid_to_context
2)  Call context_to_sid
3)  Call transition_sid
4)  Call member_sid
5)  Call change_sid
6)  Call list_sids
7)  Call load_policy
8)  Call fs_sid
9)  Call port_sid
a)  Call netif_sid
b)  Call node_sid
c)  Call fs_use
d)  Call genfs_sid
e)  Call get_user_sids
f)  display conditional bools
g)  display conditional expressions
h)  change a boolean value
i)  display constraint expressions
j)  display validatetrans expressions
m)  Show menu again
q)  Exit

Choose:
```

**Comment by pasis** [comment]
I updated my repo (https://github.com/pasis/setools-android). This issue should be fixed there. @xmikos Please, merge our repos and post version 1.3.

**Comment by xmikos** [comment]
Updates by @pasis are now merged, latest setools-android version 1.3 works on Android N.
